VAMobile/src/screens/HealthScreen/Appointments/UpcomingAppointments/UpcomingAppointmentDetails.tsx

```diff
@@ -375,46 +381,66 @@ function UpcomingAppointmentDetails({ route, navigation }: UpcomingAppointmentDe
     )
   }
 
+  const isInPersonVAAppointment =
+    appointmentType === AppointmentTypeConstants.VA && serviceCategoryName !== 'COMPENSATION & PENSION'
+
   return (
     <FeatureLandingTemplate
       backLabel={t('appointments')}
       backLabelOnPress={navigation.goBack}
       title={t('details')}
       testID="UpcomingApptDetailsTestID">
       <Box mb={theme.dimensions.contentMarginBottom}>
-        <AppointmentAlert attributes={attributes} />
-        <TextArea>
-          <AppointmentTypeAndDate attributes={attributes} isPastAppointment={false} />
-          {renderAddToCalendarLink()}
-
-          {renderVideoAppointmentInstructions()}
-
-          {renderAtHomeVideoConnectAppointmentData()}
-          <TypeOfCare attributes={attributes} />
-          <ProviderName attributes={attributes} />
-          <ClinicNameAndPhysicalLocation attributes={attributes} />
-          <AppointmentAddressAndNumber attributes={attributes} isPastAppointment={false} />
-
-          {renderAtlasVideoConnectAppointmentData()}
-          {featureEnabled('patientCheckIn') && (
-            <Box my={theme.dimensions.gutter} mr={theme.dimensions.buttonPadding}>
-              <Button onPress={() => navigateTo('ConfirmContactInfo')} label={t('checkIn.now')} />
-            </Box>
-          )}
-          <PreferredDateAndTime attributes={attributes} />
-          <PreferredAppointmentType attributes={attributes} />
-          <AppointmentReason attributes={attributes} />
-          {renderSpecialInstructions()}
-          <ContactInformation attributes={attributes} />
-          <PendingAppointmentCancelButton
+        {isInPersonVAAppointment ? (
+          <InPersonVAAppointments
             attributes={attributes}
-            appointmentID={trueAppointment?.id}
-            cancelAppointment={cancelAppointment}
-            goBack={navigation.goBack}
+            subType={
+              isAppointmentCanceled && pendingAppointment
+                ? AppointmentDetailsSubTypeConstants.CanceledAndPending
+                : isAppointmentCanceled
+                  ? AppointmentDetailsSubTypeConstants.Canceled
+                  : pendingAppointment
+                    ? AppointmentDetailsSubTypeConstants.Pending
+                    : AppointmentDetailsSubTypeConstants.Upcoming
+            }
           />
-        </TextArea>
+        ) : (
+          <Box>
```

Comment:
That is the idea. I am keeping this all here as this is what is in live and didn't want to touch any of that actual code. When all of the details screens are done this will eventually be retired.